server/src/test/java/io/crate/types/NumericTypeTest.java

```diff
@@ -87,6 +87,7 @@ public void test_implicit_cast_decimal_types_to_unscaled_numeric() {
     public void test_implicit_cast_text_types_to_numeric_with_precision() {
         assertThat(new NumericType(5, 0).implicitCast("12345")).isEqualTo(BigDecimal.valueOf(12345));
         assertThat(new NumericType(6, null).implicitCast("12345")).isEqualTo(BigDecimal.valueOf(12345));
+        assertThat(new NumericType(6, 2).implicitCast("1234.56")).isEqualTo(BigDecimal.valueOf(1234.56));
```

Comment:
Also here, why do you want to add this? Afaik it's not really related. Not completely against adding it, just want to understand the motivation.

Reply:
just another left over, apologies and thx!